apps/test-gimbal/main.cpp

```diff
@@ -0,0 +1,34 @@
+#include "mbed.h"
+
+#include "Encoder.h"
+#include "EncoderAbsolute_PWM.h"
+#include "ActuatorController.h"
+#include "CANMsg.h"
+#include "CANBuffer.h"
+#include "Servo.h"
+#include "PwmOut.h"
+
+#define TIME_ERROR_FACTOR 1
+
+// Init. Components
+
+PwmOut tilt (PB_15);
+PwmOut pan (PB_14);
+
+int max_width_us = 2.1*1000*TIME_ERROR_FACTOR,
+    min_width_us = 0.9*1000*TIME_ERROR_FACTOR;

```

Comment:
Wouldn't it be more optimal to just write 900 instead of 0.9*1000? You can always show the calculation in a comment if that's the desired purpose here.